examples/module1/lesson1/validate-it/helpers.ts

```diff
@@ -0,0 +1,17 @@
+import type { CompareProps } from './types';
+
+export const isInteger = (value: number) => {
+  return Number.isInteger(value);
+};
+
+export const isSmaller = ({ value, compareValue }: CompareProps) => {
+  return value < compareValue;
+};
+
+export const isHigher = ({ value, compareValue }: CompareProps) => {
+  return value > compareValue;
+};
```

Comment:
Mały komentarz stylistyczny - w angielskim high sugeruje wzrost, a small wielkość. Jeśli mówimy o liczbach, to wg konwencji, zazwyczaj stosuje się "isLower" albo "isGreater" bo mówimy o jednej skali i poruszaniu się "w lewo i w prawo" ;) Ot, taki detal.